api_tests/dry/sanity.js

```diff
@@ -69,6 +69,16 @@ describe("Sanity tests", () => {
             });
     });
 
+    it("[Alice] Sets appropriate CORS headers", async () => {
+        let res = await chai
```

Comment:
🔧 Should we verify that `alice.comit_node_url() !== "localhost:3000"`? 
```suggestion
        alice.comit_node_url().should.not.equal("localhost:3000");
        let res = await chai
```